suzieq/poller/controller/source/netbox.py

```diff
@@ -151,6 +164,24 @@ def _token_auth_header(self) -> Dict:
         """
         return {'Authorization': f'Token {self._data.token}'}
 
+    def _get_url_list(self) -> List[str]:
+        """Return the list of requests to execute
+
+        Returns:
+            List[str]: list of urls
+        """
+        urls = []
+        url_address = f'{self._server.protocol}://{self._server.host}:'\
+            f'{self._server.port}/api/dcim/devices/?'
+        for tag in self._data.tag:
```

Comment:
```suggestion
        for tags in self._data.tag:
```
Maybe we should use plural var name as each row might have more than one tag